platform/core/src/classes/MetadataProvider.ts

```diff
@@ -531,6 +533,11 @@ const WADO_IMAGE_LOADER = {
     let imageOrientationPatient;
     if (PixelSpacing) {
       [rowPixelSpacing, columnPixelSpacing] = PixelSpacing;
+      calibratedPixelSpacingMetadataProvider.add(instance.imageId, {
+        rowPixelSpacing: parseFloat(PixelSpacing[0]),
+        columnPixelSpacing: parseFloat(PixelSpacing[1]),
+        type: null,
```

Comment:
The type for DX should not be null - it needs to get the right internal value from the getPixelSpacingInformation object.